### ISSUE TITLE
Be able to mock websocket calls 

### ISSUE THREAD

**Issue by jkanchelov** [miragejs/miragejs#315]
I think if there is some kind of WebSocket mocking support next to the rest API would be a great feature.

**Comment by samselikoff** [comment]
Yes, I've done this before using https://github.com/thoov/mock-socket.

We're going to be adding a `server.handle(response)` API to Mirage that would make this much easier.

At the least, we should add an advanced guide or cookbook entry to the site to explain how to do this today.

**Comment by samselikoff** [comment]
https://github.com/miragejs/site/issues/297
